### ISSUE TITLE
Code review

### ISSUE THREAD

**Issue by Tianyi94** [EricZhengAZ/Lable-Track1EC601#1]
Nice job! It seems like that you already make an APP and it does work. The code is well organized and easy to be imported. The readme is clear and enough comments are given.
However, there still some things you can to make the project better. Here is my advice.
1. The name of this repository is Lable-Track1EC601. I think it should be "label" rather than "lable". You can change it in the setting part.
2.  I ran your APP in the Android Studio and the virtual device is Nexus 5X API 27. The problem I found out is that when I rotate the device, the APP will stop and quit. Maybe you could find this bug and fix it.
3. When I opened your APP for the first time, it asked me to register. I tried using google account to sign in but it not works.
![qq 20171113164723](https://user-images.githubusercontent.com/31803844/32737605-87df18c0-c868-11e7-9ffe-e8404d31f9be.png)
4. Now the APP will judge what is showed in front of the camera and give the Amazon link of the item that has the highest possibility to be. However, this APP may make mistakes and do the wrong judgment. To improve the user-experience, I strongly suggest you list all the things that have the top three possibilities and allow the user to select it.
5. I know it's hard to learn and label all the things, so maybe you should write what kinds of objects you can label.
In summary, the organizations and comments of the code are good. The readme is clear and the APP is able to be imported. There are some little problems but the main idea of this APP is reached.
Hope your APP will be the best-seller in the future.
